lib/gollum-lib/markups.rb

```diff
@@ -6,7 +6,7 @@ class Markup
         Kramdown::Document.new(content, :auto_ids => false, :input => "markdown").to_html
     }
 
-    register(:markdown,  "Markdown", :regexp => /md|mkdn?|mdown|markdown/)
+    register(:markdown,  "Markdown", :regexp => /md|mkdn?|mdown|markdown/, :h1 => /^(#\s+)(\w+.*)/)
```

Comment:
Thanks @SkyCrawl. I say we only activate the editable sections capability for markups that have an h1 regex defined. So if we can't come up with a good regex for org and rst, that's too bad for them. :)